Makefile

```diff
@@ -47,7 +47,14 @@ starknet_programs/%.json: starknet_programs/%.cairo
 compile-abi:
 	. starknet-venv/bin/activate && cd starknet_programs/ && starknet-compile fibonacci.cairo \
 		--output fibonacci_compiled.json \
-		--abi fibonacci_abi.json
+		--abi fibonacci_abi.json \
+		&& starknet-compile constructor.cairo \
+		--output constructor_compiled.json \
+		--abi constructor_abi.json \
```

Comment:
This doesn't scale. We could instead add the `--abi` flag to `starknet_programs/%.json` targets.

Reply:
I thought about it but I'm not sure it would be useful to have every single Cairo contract ABI. I think for now these two are the only ABI we need for testing purposes. Should we add them anyways?

Reply:
This would simplify the `Makefile` and make the next abi test addition painless. I think it's a good enough tradeoff. Even if it's not useful to have them, it's _easier_ to just have them. They should be cached by the CI anyways (this doesn't happen currently though 😞).